.ci/install_crio.sh

```diff
@@ -55,23 +52,23 @@ then
 		else
 			crio_version=$(get_version "externals.crio.meta.openshift")
 		fi
+		crictl_version=$(get_version "externals.crio.meta.crictl")
+		crictl_tag_prefix=""
 	else
 		crio_version=$(get_version "externals.crio.version")
+		crictl_version=$(get_test_version "externals.critools.version")
+		crictl_tag_prefix="v"
 	fi
-
-	git fetch
-	git checkout "${crio_version}"
 fi
 
-echo "Get CRI Tools"
-critools_repo="${kubernetes_sigs_org}/cri-tools"
-go get "$critools_repo" || true
-pushd "${GOPATH}/src/${critools_repo}"
-critools_version=$(grep "ENV CRICTL_COMMIT" "${GOPATH}/src/${crio_repo}/Dockerfile" | cut -d " " -f3)
-git checkout "${critools_version}"
-go install ./cmd/crictl
-sudo install "${GOPATH}/bin/crictl" /usr/bin
-popd
+echo "Installing CRI Tools"
+crictl_url=https://github.com/kubernetes-sigs/cri-tools/releases/download/v$crictl_version/crictl-$crictl_tag_prefix$crictl_version-linux-"$(${cidir}/kata-arch.sh -g)".tar.gz
+curl -Ls "$crictl_url" | tar xfz - -C /usr/local/bin
```

Comment:
can you please add `sudo` here?

Reply:
Yes, added it. I thought it was already there :thinking:

Reply:
yep, don't worry, I think this should be fine now 😄 
/test